packages/functional-tests/pages/settings/components/unitRow.ts

```diff
@@ -107,8 +107,8 @@ export class TotpRow extends UnitRow {
 
 export class ConnectedServicesRow extends UnitRow {
   async services() {
-    await this.page.waitForSelector('#service');
-    const elements = await this.page.$$('#service');
+    await this.page.waitForSelector('.my-1.settings-connected-service');
+    const elements = await this.page.$$('.my-1.settings-connected-service');
```

Comment:
Do we need the added specificity of the `.my-1` prefix here?
Or should we use the exported `selectors.SETTINGS.CONNECTED_SERVICES.SERVICE` const, which seems to be:
```js
SERVICE: '[data-testid=settings-connected-service]',
```

Although that'd change the logic from waiting on `className` to `data-testid`:
```diff
-className="my-1 settings-connected-service"
+data-testid="settings-connected-service"
```

Which I guess I've never looked into. Can we have multiple, duplicate `data-testid`s in a document? I guess we did before, but I'm not finding an official answer on whether it's better to be unique or not.

Also, since this is in "/functional-tests/" and a super-nit, so feel free to laugh and ignore.

Reply:
Agreed. Updated to maintain consistency with the rest of the file, which uses `data-testid`.